src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/workflow.html.twig

```diff
@@ -23,6 +145,54 @@
                 flowchart: { useMaxWidth: false },
                 securityLevel: 'loose',
             });
+
+            {% for name, data in collector.workflows %}
+                window.showNodeDetails{{ collector.hash(name) }} = function (node) {
+                    const map = {{ data.listeners|json_encode|raw }};
+                    showNodeDetails(node, map);
+                };
+            {% endfor %}
+
+            const showNodeDetails = function (node, map) {
+                const dialog = document.getElementById('detailsDialog');
+                dialog.querySelector('tbody').innerHTML = '';
+                for (const [eventName, listeners] of Object.entries(map[node])) {
+                    listeners.forEach(listener => {
+                        const row = document.createElement('tr');
+                        let listenerDetails = `<code>${listener.title}</code>`;
+                        if (listener.file) {
+                            listenerDetails = `<a href="${listener.file}">${listenerDetails}</a>`;
+                        }
+                        if (typeof listener.guardExpressions === 'object') {
+                            console.log('XXX"');
+                            listenerDetails = `${listenerDetails} (guard expressions: <code>${listener.guardExpressions.join('</code>, <code>')})</code>`;
+                        }
+                        row.innerHTML = `
+                            <td><code>${eventName}</code></td>
+                            <td>${listenerDetails}</td>
+                        `;
+                        dialog.querySelector('tbody').appendChild(row);
+                    });
+                };
+
+                dialog.addEventListener('click', (e) => {
```

Comment:
This listener should not be added again each time the modal is shown, otherwise we will have multiple ones when we use the dialog multiple times, which might do weird things and will definitely leak